packages/kotti-input-number/src/InputNumber.vue

```diff
@@ -137,70 +148,91 @@ export default {
 	},
 	watch: {
 		value: {
-			handler(newNumber) {
+			handler(newNumber, oldNumber) {
 				const newString = toString(newNumber)
-				const shouldUpdate = this.currentValueNumber !== toNumber(newString)
+				const truncatedNumber = toNumber(newString)
```

Comment:
this is just refactored code with different naming, 
and two error handlers, obvs